README.md

```diff
@@ -9,6 +9,14 @@ The Node.js project uses an [open governance model](./GOVERNANCE.md). The
 
 **This project has a [Code of Conduct][].**
 
+In addition to the code of conduct requirements, contributors to the
```

Comment:
Can this text be added to the code of conduct instead?

Reply:
We discussed that, and basically the answer is no. The Code of Conduct needs to be the exact language that it is because of I think legal reasons, or it needs to follow the GitHub or OpenJS Foundation standard, I forget exactly.